internal/inspectimage/bom_display.go

```diff
@@ -14,20 +14,42 @@ type BOMDisplay struct {
 	LocalErr  string            `json:"local_error,omitempty" yaml:"localError,omitempty"`
 }
 
+type BOMWithExtensionDisplay struct {
+	Remote    []BOMEntryWithExtensionDisplay `json:"remote" yaml:"remote"`
+	Local     []BOMEntryWithExtensionDisplay `json:"local" yaml:"local"`
+	RemoteErr string                         `json:"remote_error,omitempty" yaml:"remoteError,omitempty"`
+	LocalErr  string                         `json:"local_error,omitempty" yaml:"localError,omitempty"`
+}
+
 type BOMEntryDisplay struct {
 	Name      string                 `toml:"name" json:"name" yaml:"name"`
 	Version   string                 `toml:"version,omitempty" json:"version,omitempty" yaml:"version,omitempty"`
 	Metadata  map[string]interface{} `toml:"metadata" json:"metadata" yaml:"metadata"`
 	Buildpack dist.ModuleRef         `json:"buildpacks" yaml:"buildpacks" toml:"buildpacks"`
 }
 
+type BOMEntryWithExtensionDisplay struct {
```

Comment:
I have the feeling that this should name `BOMEntryWithExtension`? I mean, a Display contains entries of this struct, right?